### ISSUE TITLE
Add query information

### ISSUE THREAD

**Issue by Matix-Media** [AlphaSoundZ/apollo#17]
Add query information to the response.

Example:  
`/api/v5/users?query=Max%20Mustermann`
```jsonc
{
    "query": "Max Mustermann",
    "order": {...},
    "page": {...},
    "users": [...]  // Or data or something idk
}
```

**Comment by AlphaSoundZ** [comment]
didn't add the feature

**Comment by AlphaSoundZ** [comment]
Response:

```json
"search": {
    "strict": false,
    "query": "test"
}
```

**Comment by AlphaSoundZ** [comment]
not finished, does not work properly

**Comment by AlphaSoundZ** [comment]
Closed with: [4ab1c16](https://github.com/AlphaSoundZ/apollo/commit/4ab1c16c26e7b035ee94113e0bbdf097a8e0bc74)

Example: 
```GET api/v5/user?order_by=id&order_strategy=DESC&page=0&size=5&query=Test```

```
{
    "response": "SUCCESS",
    "message": "Suche erfolgreich",
    "order": {
        "by": "id",
        "strategy": "DESC"
    },
    "page": {
        "total": 1,
        "current": 0,
        "size": 5,
        "total_rows": 2
    },
    "search": {
        "strict": false,
        "query": "Test"
    },
    "data": [...]
}
```

**Comment by AlphaSoundZ** [comment]
> Closed with: [4ab1c16](https://github.com/AlphaSoundZ/apollo/commit/4ab1c16c26e7b035ee94113e0bbdf097a8e0bc74)
> 
> Example: `GET api/v5/user?order_by=id&order_strategy=DESC&page=0&size=5&query=Test`
> 
> ```
> {
>     "response": "SUCCESS",
>     "message": "Suche erfolgreich",
>     "order": {
>         "by": "id",
>         "strategy": "DESC"
>     },
>     "page": {
>         "total": 1,
>         "current": 0,
>         "size": 5,
>         "total_rows": 2
>     },
>     "search": {
>         "strict": false,
>         "query": "Test"
>     },
>     "data": [...]
> }
> ```

Possible optimization:
Write function for generating the parts of the response that keep the same every time.
For example:
```
{
    "order": [...],
    "search": [...],
}
